clickstream/src/main/java/software/aws/solution/clickstream/ClickstreamAnalytics.java

```diff
@@ -111,7 +111,21 @@ public static void setUserId(String userId) {
     }
 
     /**
-     * get clickstream configuration
+     * Enable clickstream SDK.
+     */
+    public static void enable() {
+        Amplify.Analytics.enable();
```

Comment:
Is it safe to invoke multiple times even if the SDK is enabled?

Reply:
Yes, this is unsafe, after integration testing it was found that the `Handler` will be initialized multiple times. I added state to control it and now it will be safe to call enable or disable multiple times.